src/main/java/com/spartronics4915/frc2019/Constants.java

```diff
@@ -30,9 +30,9 @@
     public static final double kRobotLinearInertia = 60.0; // kg TODO tune
     public static final double kRobotAngularInertia = 10.0; // kg m^2 TODO tune
     public static final double kRobotAngularDrag = 12.0; // N*m / (rad/sec) TODO tune
-    public static final double kDriveVIntercept = 1.055; // V
-    public static final double kDriveKv = 0.135; // V per rad/s
-    public static final double kDriveKa = 0.012; // V per rad/s^2
+    public static final double kDriveVIntercept = 1.115; // V
```

Comment:
probably premature, but we theoretically might need two of each of these.

Reply:
That's my next step... We will also need to instantiate two `DcMotorTransmissions`.